app/styleguide/components/s-palette-list.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div :class="b()">
+    <s-palette-item
+      v-for="palette in palettes"
+      :palette="palette"
+      :colors="colors[palette]"
+      :key="palette"/>
+  </div>
+</template>
+
+<script>
+
+  import _colors from '../../setup/js/_color';
```

Comment:
- [ ] Please don't use underline in JS. In SCSS it is common convention to mark include only files with an underline at the beginning. Don't apply this to JS.